tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -1917,6 +1927,55 @@ void import_export_public_key(data_t *data,
         TEST_MEMORY_COMPARE(expected_public_key->x, expected_public_key->len,
                             exported, exported_length);
     }
+
+    /* Adjust expected_status for interruptible export public-key.
+     * Interruptible export public-key is only supported for ECC key pairs and even
+     * for those only when MBEDTLS_ECP_RESTARTABLE is on.
+     */
+    if ((PSA_KEY_TYPE_IS_KEY_PAIR(type) || PSA_KEY_TYPE_IS_PUBLIC_KEY(type)) &&
+        !PSA_KEY_TYPE_IS_ECC(type)) {
+        expected_export_status = PSA_ERROR_NOT_SUPPORTED;
+    }
+
+#if !defined(MBEDTLS_ECP_RESTARTABLE)
+    expected_export_status = PSA_ERROR_NOT_SUPPORTED;
+#endif
+
+    if (PSA_KEY_LIFETIME_GET_LOCATION(psa_get_key_lifetime(&attributes)) !=
+        PSA_KEY_LOCATION_LOCAL_STORAGE) {
+        expected_export_status = PSA_ERROR_NOT_SUPPORTED;
+    }
+
+    status = psa_export_public_key_iop_setup(&export_key_operation, key);
+    TEST_EQUAL(status, expected_export_status);
```

Comment:
When `expected_export_status` is `PSA_ERROR_BUFFER_TOO_SMALL`, I'd expect setup to succeed, and complete to fail. I think we're missing test cases for these: we have them only for RSA.

That's partly preexisting: we should have had buffer-too-small test cases for ECC key pairs and ECC public keys, not just for RSA. But now it's more important, because we have new code paths for ECC keys. So please add these test cases (and, I guess, adjust the test code accordingly).